src/components/displaySvg/finalSvgDisplay.scss

```diff
@@ -0,0 +1,69 @@
+.display-page-div{
+    padding: 171px;
+    width: 373px; 
+}
+
+  a:hover {
+   background-color: yellow;
+   fill:yellow;
+ }
+
+ svg{
+    max-height: 106px;
+    max-width: fit-content;
+    margin-top: 94px;
+    background-color: white;
+ }
+  
+ .display-svgs-border-match {
+    max-height: 381px;
+    border: outset;
+    border-color: green;
+}
+
+ .display-original-optimised{
+    overflow-y: scroll;
+    max-height: 571px;
+    border: outset;
+    border-color: green;
+ }
+
+ .button-background{
+    background-color: grey;
+    height: 171px;   
+ }
+
+ .loader-text{
+      margin-top: 210px;
+ }
+
+ .loader {
+   border: 16px solid greenyellow;
+    border-radius: 114%;
```

Comment:
why is this styling number so high

Reply:
no, like seriously

Reply:
px work better than % when trying to apply roundness with css

Reply:
Okay I did not know that, thanks let me change into %

Reply:
you mean px?

Reply:
lol yes, actualluy I have already changed it into px.